cmake/Platform/Common/Install_common.cmake

```diff
@@ -635,7 +635,7 @@ endif()
         string(APPEND builtinpackages "${builtin_packages_inject_code}\n")
     endif()
 
-    set(permutation_builtin_file ${CMAKE_CURRENT_BINARY_DIR}/cmake/3rdParty/Platform/${PAL_PLATFORM_NAME}/${LY_BUILD_PERMUTATION}/BuiltInPackages_${PAL_PLATFORM_NAME_LOWERCASE}.cmake)
+    set(permutation_builtin_file ${CMAKE_CURRENT_BINARY_DIR}/cmake/3rdParty/Platform/${PAL_PLATFORM_NAME}/${LY_BUILD_PERMUTATION}/BuiltInPackages_${PAL_PLATFORM_NAME_LOWERCASE}${LY_HOST_ARCHITECTURE_NAME_EXTENSION}.cmake)
```

Comment:
Line [602-609](https://github.com/o3de/o3de/blob/ddfd53fda679ada1b89f07037ab5967d8d6c5817/cmake/Platform/Common/Install_common.cmake#L602-L609) also needs to be updated to take the architecture into account.